vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -83,7 +83,7 @@ def build(self):
         return dbc.Container(
             id="dashboard_container_outer",
             children=[
-                html.Div(id=f"vizro_version_{vizro.__version__}"),
+                html.Div(vizro.__version__, id="vizro_version", hidden=True),
```

Comment:
@huong-li-nguyen this is a very small and insignificant change but feels like a much tidier way to do it since it separates the label `vizro_version` from the data itself: `<div id="vizro_version" hidden>0.1.6</div>`.

Reply:
Makes sense!